python_modules/dagster/dagster_tests/daemon_tests/test_sensor_run.py

```diff
@@ -222,12 +227,27 @@ def my_pipeline_success_sensor(context):
     assert isinstance(context.instance, DagsterInstance)
 
 
+@multi_job_sensor(jobs=[the_job, config_graph.to_job()])
+def two_job_sensor(context):
+    counter = int(context.cursor) if context.cursor else 0
+    if counter % 2 == 0:
+        yield RunRequest(run_key=str(counter), job_name="the_graph")
+    else:
+        yield RunRequest(
+            run_key=str(counter),
+            job_name="config_graph",
+            run_config={"solids": {"config_solid": {"config": {"foo": "blah"}}}},
+        )
```

Comment:
There something odd about pointing at the job objects in the decorator, and then aligning with their name string property from the sensor. I guess we could update the examples to do `job_name=the_job.name` or something.

When we discussed this in the past I pitched having targets being a `dict` and then aligning on the key. Awkward in its own way, but has the upside of preventing a change to the graph/job name breaking the sensor at run time if its using string literals for `job_name`

Reply:
An extra layer of indirection adds a lot of pain / opportunities for error, so I think we should be pretty confident it's necessary if we're going to add it.  If someone is worried about the job name changing, they could do `RunRequest(job_name=my_job.name)`.

Reply:
Maybe `the_job.create_run_request(...)`  could be interesting, might be able to give a better error experiences.

The fundamental awkwardness is some amount of duping since we want the ahead of time declaration and then need to align amongst the multiple targets.

Reply:
I found `RunRequest(job_name=job.name)` to be pretty good... not sure why, but I feel hesitant to create methods on `JobDefinition` that are thin wrappers around `RunRequest`